tests/spec/w3c/headers-spec.js

```diff
@@ -557,6 +557,105 @@ describe("W3C — Headers", () => {
     });
   });
 
+  describe("valid level config", () => {
+    it("updates relevant locations with the level when level is the integer 0", async () => {
+      const body = `
+        <h1 id="title">Spec <code>Marked</code> Up</h1>${makeDefaultBody()}`;
+      const ops = makeStandardOps({}, body);
+      const newProps = { level: 0, specStatus: "REC", shortName: "abc" };
+      Object.assign(ops.config, newProps);
+
+      const doc = await makeRSDoc(ops);
+
+      const h1Elem = doc.querySelector("h1#title");
+      // html is not escaped
+      expect(h1Elem.childNodes[1].localName).toBe("code");
+      expect(h1Elem.textContent).toBe("Spec Marked Up Level 0");
+      expect(doc.title).toBe("Spec Marked Up Level 0");
+
+      const terms = doc.querySelectorAll("dt");
+      expect(terms[0].textContent).toBe("This version:");
+      expect(terms[0].nextElementSibling.localName).toBe("dd");
+      expect(terms[0].nextElementSibling.textContent).toContain("REC-abc-0");
+      expect(terms[1].textContent).toBe("Latest published version:");
+      expect(terms[1].nextElementSibling.localName).toBe("dd");
+      expect(terms[1].nextElementSibling.textContent).toContain("TR/abc-0/");
+    });
+
+    it("updates relevant locations with the level when level is an integer greater than 0", async () => {
+      const body = `
+      <h1 id="title">Spec <code>Marked</code> Up</h1>${makeDefaultBody()}`;
+      const ops = makeStandardOps({}, body);
+      const newProps = { level: 9870, specStatus: "REC", shortName: "xyz" };
+      Object.assign(ops.config, newProps);
+
+      const doc = await makeRSDoc(ops);
+
+      const h1Elem = doc.querySelector("h1#title");
+      // html is not escaped
+      expect(h1Elem.childNodes[1].localName).toBe("code");
+      expect(h1Elem.textContent).toBe("Spec Marked Up Level 9870");
+      expect(doc.title).toBe("Spec Marked Up Level 9870");
+
+      const terms = doc.querySelectorAll("dt");
+      expect(terms[0].textContent).toBe("This version:");
+      expect(terms[0].nextElementSibling.localName).toBe("dd");
```

Comment:
We can probably use 
```
version = terms[0].querySelector(':scope > dd')
version.textContent === ...
```

What I mean is these tests can be simplified a bit to not involve nextSibling like traversals...

Reply:
Never mind.. this looked bad only on GitHub app while reviewing.